src/sentry/web/frontend/error_page_embed.py

```diff
@@ -139,6 +140,9 @@ def dispatch(self, request):
                     comments=report.comments,
                     date_added=timezone.now(),
                 )
+
+            user_feedback_received.send(project=report.project, group=report.group, sender=self)
```

Comment:
`report.project` is called on L135. `report.group` is called on L122.
`group` isn't necessary, if we'd rather not pass it though.